var/spack/repos/builtin/packages/phist/package.py

```diff
@@ -79,6 +81,7 @@ class Phist(CMakePackage):
     depends_on('cmake@3.8:', type='build')
     depends_on('blas')
     depends_on('lapack')
+    depends_on('python@3', when='@1.7:')
```

Comment:
This should be `python@3:`, otherwise it will try to download and install exactly version 3.